README.md

```diff
@@ -1,12 +1,12 @@
 # SLI/SLO flutter sample
```

Comment:
How about call `user experience SLI/SLO` to match the slides?

Reply:
great idea.
```suggestion
# SLI/SLO flutter sample for measuring "user experience SLI/SLO"
```